android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java

```diff
@@ -2104,8 +2145,8 @@ protected Object getValue(JSONObject object) throws JSONException {
         Object value;
 
         if (object.has(JsonFormConstants.VALUE)) {
-            value = object.opt(JsonFormConstants.VALUE);
-
+            value = Utils.getValueFromTranslatedObject(object);
+//            value = object.opt(JsonFormConstants.VALUE);
```

Comment:
@SebaMutuku Please remove the commented out code

Reply:
Done